### ISSUE TITLE
What is motivation of using sigmoid at last layer?

### ISSUE THREAD

**Issue by mrgloom** [ZFTurbo/ZF_UNET_224_Pretrained_Model#1]
What is motivation of using sigmoid at last layer?
https://github.com/ZFTurbo/ZF_UNET_224_Pretrained_Model/blob/master/zf_unet_224_model.py#L105

**Comment by ZFTurbo** [comment]
It's suitable for the case where masks in different channels can overlap. For single output channel it's the same as softmax. Also softmax can't be used directly for 2D output in Keras.

**Comment by mrgloom** [comment]
Looks like reshape is not needed in case of softmax+categorical_crossentropy too, it just applied by last dimension.
